### PR TITLE
Clean up the English some

### PR DESCRIPTION
**Pull request by steveno**

### CODE REVIEW

File: docs/building.rst [hunk 1]
```diff
@@ -10,19 +10,18 @@ Building ctags
 Build system add possibility to change program name
 ---------------------------------------------------------------------
 
-As on some systems (e.g. BSD) there is a 'ctags' program in the base
-system it's somewhat inconvenient to have the same name for universal-ctags
-During ``configure`` you can now change the output executable name.
+As on some systems (e.g. BSD), there is alreayd a 'ctags' program in the base
```
Comment (vhda) [inline docs/building.rst]: "already"

Comment (vhda) [inline docs/building.rst]: Or maybe: On some systems, like BSD, there is already a 'ctags' program in the base

File: docs/building.rst [hunk 2]
```diff
@@ -10,19 +10,18 @@ Building ctags
...
 
-Add a prefix 'ex' which will result in 'ctags' transformed into 'exctags'
+Add a prefix 'ex' which will result in 'ctags' being transformed into 'exctags'
 ::
 
 	$ ./configure --program-prefix=ex
 
-Completely change program name, in this case it's important to remember
-there is also 'etags' along 'ctags'
+If you would like to completely change program name please remember in this case
```
Comment (vhda) [inline docs/building.rst]: Since we're improving, please avoid the use of "you" in the sentences.

**Comment by steveno** [comment]
@vhda I can clean it up and resubmit. No problem!

**Comment by vhda** [comment]
@steveno have you `git push -f` the changes? Still don't see the updated commits in GitHub.

**Comment by steveno** [comment]
@vhda I have not made your suggested updates. I'm sorry. I will do it probably this weekend if that's OK.

**Comment by vhda** [comment]
There is no pressure, clearly I misunderstood your last statement :)

**Pull request closed without merging**